include/CLI/impl/App_inl.hpp

```diff
@@ -1317,10 +1317,22 @@ CLI11_INLINE void App::_process_requirements() {
             missing_need = opt->get_name();
         }
     }
+    for(const auto &subc : need_subcommands_) { // Process subcommands given on the commandline first
```

Comment:
So you want to check the subcommands that are needed and were given and make sure all their requirements are met first.   This is to get at the lower level errors with priority.